src/styles/Navbar.css

```diff
@@ -0,0 +1,45 @@
+.main-menu {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem;
+  margin: 0;
+  margin-bottom: 0;
+  border-bottom: 1px solid gray;
+  background: orange;
+  color: white;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+  font-weight: bolder;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: space-around;
+  width: 15rem;
+  align-items: center;
+  font-size: 1.2rem;
+}
+
+/* .menu-logo {
+  height: 5vmin;
+  pointer-events: none;
+} */
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))